Terminal.Gui/Views/Menu/Menu.cs

```diff
@@ -432,7 +432,7 @@ void AddKeyBindings (MenuBarItem menuBarItem)
 		foreach (var menuItem in menuBarItem.Children.Where (m => m != null)) {
 			KeyBindings.Add ((KeyCode)menuItem.HotKey.Value, Command.ToggleExpandCollapse);
 			KeyBindings.Add ((KeyCode)menuItem.HotKey.Value | KeyCode.AltMask, Command.ToggleExpandCollapse);
-			if (menuItem.Shortcut != KeyCode.Unknown) {
+			if (menuItem.Shortcut != KeyCode.Null) {
```

Comment:
Here I've doubt if only a Shift, Alt or Ctrl pass.

Reply:
Ok but `Alt` is acceptable, right?

Reply:
Regarding a key event with ONLY shift modifiers.... We should ensure they never get sent by drivers.

In otherwords, nobody should REALLY have to call `Key.IsValid`. 

Thus, i think this is ok.

Reply:
Also, https://github.com/gui-cs/Terminal.Gui/issues/2975 will revamp all of this...